packages/block-editor/src/components/block-list/block-crash-warning.js

```diff
@@ -9,7 +9,7 @@ import { __ } from '@wordpress/i18n';
 import Warning from '../warning';
 
 const warning = (
-	<Warning>
+	<Warning className="block-editor-block-crash-warning">
```

Comment:
I think the proper name should be:
```suggestion
	<Warning className="block-editor-block-list__block-crash-warning">
```

See: https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#naming

Reply:
Good point, now fixed 👍